overFeatWrapper/detectedObj[] detected_classes
